tests/flagd/pkg/integration/evaluation.go

```diff
@@ -7,12 +7,11 @@ import (
 	"strconv"
 
 	"github.com/cucumber/godog"
-	flagd "github.com/open-feature/go-sdk-contrib/providers/flagd/pkg"
 	"github.com/open-feature/go-sdk/pkg/openfeature"
 )
 
-func InitializeEvaluationScenario(pOptions ...flagd.ProviderOption) func(*godog.ScenarioContext) {
-	providerOptions = pOptions
+func InitializeEvaluationScenario(providerSupplier func() openfeature.FeatureProvider) func(*godog.ScenarioContext) {
```

Comment:
The test suites now take a provider factory (so you can use any provider with them).